core/deployment/src/main/java/io/quarkus/deployment/pkg/NativeConfig.java

```diff
@@ -165,7 +165,7 @@
     /**
      * If full stack traces are enabled in the resulting image
      */
-    @ConfigItem(defaultValue = "false")
+    @ConfigItem(defaultValue = "true")
     public boolean fullStackTraces;
```

Comment:
What bugs me a bit too is that now, the output when creating the native image is very scarce. I don't know if it's normal or not but we used to have a couple of Quarkus lines and now we have:
```
21:02:57,864 INFO  [org.jbo.threads] JBoss Threads version 3.0.0.Final
```
and that's it.

Reply:
I will look into this